process.md

```diff
@@ -0,0 +1,97 @@
+| :three: | Monday    | Release :ship:                                                 | Work on requirements :black_nib:                               |                                  |     |
+|         | Tuesday   | Sprint review :vertical_traffic_light:, retro :roller_coaster: | Sprint planning :bookmark_tabs:                                |                                  |     |
+|         | Wednesday |                                                                |                                                                |                                  |     |
+|         | Thursday  |                                                                |                                                                |                                  |     |
+|         | Friday    |                                                                |                                                                |                                  |     |
+| :four:  | Monday    |                                                                |                                                                | Work on requirements :black_nib: |     |
+|         | Tuesday   |                                                                |                                                                | Work on requirements :black_nib: |     |
+|         | Wednesday |                                                                |                                                                | Work on requirements :black_nib: |     |
+|         | Thursday  |                                                                |                                                                | Work on requirements :black_nib: |     |
+|         | Friday    |                                                                | QA :rotating_light:                                            | Work on requirements :black_nib: |     |
+| :five:  | Monday    |                                                                | Release :ship:                                                 | Work on requirements :black_nib: |     |
+|         | Tuesday   |                                                                | Sprint review :vertical_traffic_light:, retro :roller_coaster: | Sprint planning :bookmark_tabs:  |     |
+
+Meeting outside of sprint cycle: Gnosis Safe Team weekly every Tuesday.
+
+## Activities
+
+### Work on requirements :black_nib:
+
+- Product managers :rocket: define requirements
+- Product manager prioritizes tickets
+  - Always make sure that tickets are prioritized properly.
+- Software engineers :hammer_and_wrench: add technical details
+- UX researchers perform research and tests
+- UX/UI designers :dart: refine screens
+  - Prototypes on Invision
+  - Later stages on Zeplin
+- Software engineers :hammer_and_wrench: discuss technical details
+- Software engineers :hammer_and_wrench: estimate tickets
```

Comment:
I would make the estimation a separate activity or meeting as it is a well-defined thing, usually takes 2 hours for a month of work, so it'll take around 30-60 minutes for the sprint tasks. Requirements -> estimation -> planning

Reply:
Happy to do that. On which day should it take place? Ideally it would be close before the planning meeting. So maybe Monday or Tuesday? What do you think?

Reply:
I think, Tuesday so that meetings are grouped.

Reply:
added